doc.go

```diff
@@ -37,6 +37,50 @@ For sample applications and a tutorial, see the samples directory
 (https://github.com/google/go-cloud/tree/master/samples).
 
 
+URLs
+
+In addition to creating portable types via provider-specific constructors
+(e.g., creating a blob.Bucket using s3blob.OpenBucket), many portable types
+can also be created using a URL. The scheme of the URL specifies the provider,
+and each provider implementation has code to convert the URL into the data
+needed to call its constructor. For example, calling blob.OpenBucket with
+"s3blob://my-bucket" will return a blob.Bucket created using s3blob.OpenBucket.
+
+Each portable API package will document the types that it supports opening
+by URL; for example, the "blob" package supports Buckets, while the "pubsub"
```

Comment:
Quotations are not consistent here. It's a shame godoc doesn't let us use backticks to format fixed-width-font stuff :-/

I think the convention in godoc.org is not putting quotes... Let's stick to that?

Reply:
Done.